src/popup/components/SwapRates.vue

```diff
@@ -37,39 +37,44 @@
   </div>
 </template>
 
-<script>
-import { mapState } from 'vuex';
+<script lang="ts">
+import { computed, defineComponent } from '@vue/composition-api';
 import { camelCase } from 'lodash-es';
 import { transactionTokenInfoResolvers } from '../utils/transactionTokenInfoResolvers';
 import { FUNCTION_TYPE_DEX } from '../utils/constants';
+import { useFungibleTokens } from '../../composables';
 
 import Tokens from './Tokens.vue';
 import TokenAmount from './TokenAmount.vue';
 
-export default {
+export default defineComponent({
   components: {
     Tokens,
     TokenAmount,
   },
   props: {
     transaction: { type: Object, required: true },
```

Comment:
As this file is now using TypeScript I suggest to type the prop (probably `ITransaction`).